modules/serverless-functions-secrets-configmaps-manually.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-manually_{context}"]
+= Adding function access to secrets and config maps manually
+
+You can manually add configuration for accessing secrets and config maps to your function.
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For each secret you want to mount as a volume, add the following YAML to the `volumes` section:
```

Comment:
Can this be some level of heading (subsection for each example)? I am not sure if that's aligned with the rest of the docs. Something as I would do in markdown:

# Adding function access to secrets and config maps manually

You can manually add configuration for accessing secrets and config maps to your function.

## Mounting a secret as a volume
 For each secret you want to mount as a volume, add the following YAML to the `volumes` section:
...


## Mounting a config map as a volume
For each config map you want to mount as a volume, add the following YAML to the `volumes` section:
...

## Environment value from a key-value pair defined in a secret
...


...and so on.

Reply:
I preferred the single procedure way, but since you and Ashleigh are in favor of this, I went with the majority :)